### DISCUSSION TITLE
How to loop AB -> AC -> AD

### DISCUSSION THREAD

**Discussion by TylonHH** [smartin015/continuousprint#61] [Ideas]
Hey,
I print boxes. The Bottom (A) is every time the same. Just the lids (B, C, D) are different.
So I like to print a complete box before next. I recognized that I can't the same multiple times like A B A C A D
So I separated it into three jobs
1) AB x5
2) AC x5
3) AD x5
But I wanna print first AB complete then AC then AD instead of 5x AB then 5x AC

I kinda miss the loop function for this. Then I could set the amount to x1and print in loop (because I use an run out filament sensor).
Or is there any other suggestion?

**Comment by smartin015** [comment]
I'm currently doing a [big edit](https://github.com/smartin015/continuousprint/pull/60) which will let you add multiple of the same print to a job, and reorder as you like. When that's completed, you should be able to create a job with A B A C A D, and set it to repeat as many times as you like. 

In the current version, you could make a few copies of A and rename them (e.g. A1, A2, A3). CPQ will treat them as separate sets in that case, but it is a bit of a hack.
